docs/user-guide/admin-cli.md

```diff
@@ -9,14 +10,341 @@ by the Docker user via the Docker CLI. However, the Docker CLI is limited in wha
+```
+
+##### Help
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access set -h
+usage: vmdkops_admin.py tenant access set [-h]
+                                          [--volume-totalsize Num{MB,GB,TB} - e.g. 2TB]
+                                          [--volume-maxsize Num{MB,GB,TB} - e.g. 2TB]
+                                          [--volume-maxcount VOLUME_MAXCOUNT]
+                                          [--rm-rights create,delete,mount,all]
+                                          [--add-rights create,delete,mount,all]
+                                          --name NAME --datastore DATASTORE
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --volume-totalsize Num{MB,GB,TB} - e.g. 2TB
+                        Maximum total size of all volume that can be created
+                        on the datastore for this tenant
+  --volume-maxsize Num{MB,GB,TB} - e.g. 2TB
+                        Maximum size of the volume that can be created
+  --volume-maxcount VOLUME_MAXCOUNT
+                        Maximum number of volumes to create on the datastore
+                        for this tenant
+  --rm-rights create,delete,mount,all
+                        Datastore access Permissions removed: Choices =
+                        ['create', 'delete', 'mount', 'all']
+  --add-rights create,delete,mount,all
+                        Datastore access Permissions granted: Choices =
+                        ['create', 'delete', 'mount', 'all']
+  --name NAME           Tenant name
+  --datastore DATASTORE
+                        Datastore name
+```
+### List
+
+List existing tenants, the datastores tenants have access to and the VMs assigned.
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+Uuid                                  Name     Description  Default_datastore  VM_list
+------------------------------------  -------  -----------  -----------------  ---------------------------
+dc17b3b1-1f4d-4f8b-a6f4-3dba064ba88f  tenant1               default_ds         photon.vsan.1,photon.vsan.2
+5afa4767-ecd8-489e-9d97-5c46ae50d3fc  DevAppX               default_ds         ubuntu
+```
+#### Help
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+Uuid                                  Name     Description  Default_datastore  VM_list
+------------------------------------  -------  -----------  -----------------  ---------------------------
+dc17b3b1-1f4d-4f8b-a6f4-3dba064ba88f  tenant1               default_ds         photon.vsan.1,photon.vsan.2
+5afa4767-ecd8-489e-9d97-5c46ae50d3fc  DevAppX               default_ds         ubuntu
+
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls -h
+usage: vmdkops_admin.py tenant ls [-h]
+
+optional arguments:
+  -h, --help  show this help message and exit
+```
+
+### Virtual Machine
```

Comment:
I think it would be better to move the AdminCLI to add/remove virtual machine before those commands to manipulate datastore access privileges.